vms/platformvm/api/static_service.go

```diff
@@ -36,30 +33,27 @@ var (
 	errValidatorAlreadyExited = errors.New("validator would have already unstaked")
 	errStakeOverflow          = errors.New("validator stake exceeds limit")
 
-	_ utils.Sortable[UTXO] = UTXO{}
+	_ utils.Sortable[Allocation] = Allocation{}
 )
 
-// StaticService defines the static API methods exposed by the platform VM
-type StaticService struct{}
-
-// UTXO is a UTXO on the Platform Chain that exists at the chain's genesis.
-type UTXO struct {
-	Locktime json.Uint64 `json:"locktime"`
-	Amount   json.Uint64 `json:"amount"`
-	Address  string      `json:"address"`
-	Message  string      `json:"message"`
+// Allocation is a UTXO on the Platform Chain that exists at the chain's genesis
```

Comment:
Why the renaming?

Reply:
because of type conflict. `platformvm/genesis` has UXTO already. Since this type is specifically for creating the utxo when building genesis so I renamed it to be a bit more clear.